### ISSUE TITLE
Official dataset split?

### ISSUE THREAD

**Issue by adityapatel149** [aras62/PIE#38]
The code in utilities/pie_data.py on line 90 specifies dataset split as 

{'train': ['set01', 'set02', 'set04'],
                          'val': ['set05', 'set06'],
                          'test': ['set03'],
                          'all': ['set01', 'set02', 'set03',
                                  'set04', 'set05', 'set06']}

However, most papers published used the following split:

train: set01, set02, set06
val: set04, set05
test: set03

Can you please clarify which datasplit is the official one? Does it matter which split I use given set03 remains test set?

Thank you

**Comment by ykotseruba** [comment]
The official split is in this repo. That's what we used in our models and benchmarks. Set 04 is a lot larger than set05 so it will affect the amount of training data. 
Could you please share links to a few papers that use the other test split?

**Comment by adityapatel149** [comment]
Thank you for the clarification! I will be using the split provided in the repo.

These papers used set01, set02 and set06 for training:

https://openaccess.thecvf.com/content/WACV2021/papers/Kotseruba_Benchmark_for_Evaluating_Pedestrian_Action_Prediction_WACV_2021_paper.pdf

https://arxiv.org/pdf/2301.05796

https://link.springer.com/article/10.1007/s00138-025-01751-3

**Comment by ykotseruba** [comment]
Thanks for the links. 

The first paper is ours and there is a typo in the text, we used the official split in our code (https://github.com/ykotseruba/PedestrianActionBenchmark). 

The other two papers I cannot comment on.
